packages/loopring_v3/package.json

```diff
@@ -77,6 +77,8 @@
     "pretty-quick": "^1.11.1",
     "solc": "^0.5.11",
     "solidity-coverage": "^0.5.11",
+    "solium": "^1.2.5",
+    "solium-plugin-security": "^0.1.1",
```

Comment:
I don't think this is necessary any more:
- https://github.com/duaraghav8/solium-plugin-security#installation